var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -22,6 +22,7 @@ class Petsc(Package, CudaPackage, ROCmPackage):
 
     version("main", branch="main")
 
+    version("3.19.3", sha256="008239c016b869693ec8e81368a0b7638462e667d07f7d50ed5f9b75ccc58d17")
     version("3.19.1", sha256="74db60c53c80b48d5c39e07bc39a883ecced88b9f24a5de17cf6f485a903e120")
```

Comment:
Can you also add 3.19.2 here.

Also the same for py-petsc4py. Its best to update both at the same time.

Thanks for adding these in!

Reply:
I've added 3.19.2 and also added the matching versions for py-petsc4py.